lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -143,9 +143,9 @@ <h3 class="hd hd-3 title">${_("Certificate Generation History")}</h3>
             <table>
                 <thead>
                 <tr>
-                    <td class="task-name"></td>
-                    <td class="task-date"></td>
-                    <td class="task-details"></td>
+                    <th class="task-name" scope="col">${_('Task name')}</th>
+                    <th class="task-date" scope="col">${_('Date')}</th>
```

Comment:
I'm mostly concerned about the line of code referencing browseBtn-bulk-exc-students. I wanted to verify that works, as I don't see any corresponding HTML file change.